### PR TITLE
Topic card view group list

### PR DESCRIPTION
**Pull request by CHenrie**

### CODE REVIEW

**Comment by CHenrie** [comment]
@jbest84  Please Review

File: src/Views/Account/List.js [hunk 1]
```diff
@@ -149,7 +156,31 @@ define('Mobile/SalesLogix/Views/Account/List', [
...
+                label: 'Touched',
+                onApply: function(entry, parent) {
+                    this.isEnabled = parent.hasBeenTouched(entry);
+                }
+            }]
+            );
+        },
+        hasBeenTouched:function(entry){
+        if (entry['ModifyDate']) {
+            var modifydDate = Convert.toDateFromString(entry['ModifyDate']);
```
Comment (jbest84) [inline src/Views/Account/List.js]: Multiple vars

File: src/Views/Activity/List.js [hunk 2]
```diff
@@ -4,26 +4,56 @@
...
-
+        itemIcon: 'content/images/icons/ContactProfile_48x48.png',
         //Templates
+        //Card View 
+        itemColorClassTemplate: new Simplate([
+           '{%: $$.activityColorClassByType[$.Type] || $$.itemColorClass  %}'
+        ]),
+        //Card View 
+        itemTabValueTemplate: new Simplate([
+       //'{%: $$.activityTextByType[$.Type] %}'
```
Comment (jbest84) [inline src/Views/Activity/List.js]: Remove commented out code

File: src/Views/_CardLayoutListMixin.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+            this.createIndicatorLayout();
+        },
+
+        createIndicatorLayout: function() {
+            return this.itemIndicators || {};
+        },
+        createIndicators: function(topIndicatorsNode, bottomIndicatorsNode, indicators, entry) {
+            var indicatorTemplate, indicator, options, indicatorHTML;
+            for (var i = 0; i < indicators.length; i++) {
```
Comment (jbest84) [inline src/Views/_CardLayoutListMixin.js]: Move var i declaring up one

File: src/Views/_CardLayoutListMixin.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                        }
+                    }
+                }
+            }
+        },
+
+        onApplyRowTemplate: function(entry, rowNode) {
+
+            this.applyRowIndicators(entry, rowNode);
+            //this.applyRowExt(entry);
```
Comment (jbest84) [inline src/Views/_CardLayoutListMixin.js]: Remove commented out code

**Comment by jbest84** [comment]
@CHenrie Accept

**Pull request merged**